tensorflow_serving/model_servers/main.cc

```diff
@@ -219,6 +221,11 @@ int main(int argc, char** argv) {
   core_config.aspired_version_policy =
       std::unique_ptr<AspiredVersionPolicy>(new EagerLoadPolicy);
 
+  // Override default file polling interval
+  if (file_system_polling_interval > 0) {
+    core_config.file_system_poll_wait_seconds = file_system_polling_interval;
```

Comment:
I'd prefer to give the flag the same name as the config variable, to minimize confusion. (Even if you may not like the name as much, I feel it's better to keep it consistent across the layers of the system.)